api/alias_api/__init__.py

```diff
@@ -215,7 +216,6 @@ def import_alias_api():
         )
 
     # Add the newly created module oject to sys.modules and remap the globals accessor to point at our new module
-    sys.modules["alias_api"] = alias_api
     globals()["alias_api"] = sys.modules["alias_api"]
```

Comment:
While we're here, I think we should remove this line that adds the api to the global scope of the program. The api module should be accessed through import only, and should not need this.

Reply:
hey @staceyoue , looks like as soon as you import a module, it is added to globals()
here, I followed the same logic than for [sgtk](https://github.com/shotgunsoftware/tk-core/blob/master/python/sgtk/__init__.py), to be sure that the module we try to access is the correct module we've built

Reply:
I was reading more into why you would add the module to `globals()` and it seems like the main benefit is that it allows you to directly use the module attributes instead of having to use the dot notation. In the case of the api, this doesn't really matter to us (at least the moment), but probably doesn't hurt to leave it as is.